Userland/Services/WindowServer/Menu.cpp

```diff
@@ -178,7 +178,10 @@ void Menu::draw()
     m_theme_index_at_last_paint = MenuManager::the().theme_index();
 
     VERIFY(menu_window());
+    //We need to check that the application menu is not null before we can draw it
+    if(menu_window()->backing_store() == nullptr) return; 
     VERIFY(menu_window()->backing_store());
```

Comment:
You can get rid of this VERIFY call now

Reply:
Yes, it's just an assertion.
Your code is not properly formatted, run clang-format [(link to .clang-format file)](https://github.com/SerenityOS/serenity/blob/master/.clang-format) on it to fix the issue.
Try adding a whitespace at the beginning of your comment and in between `if` and the opening brace.